addons/account/wizard/account_move_reversal.py

```diff
@@ -59,13 +93,19 @@ def _compute_from_moves(self):
 
     def _prepare_default_reversal(self, move):
         reverse_date = self.date if self.date_mode == 'custom' else move.date
+        if self.journal_id:
+            jouranl_id = self.journal_id.id
+        elif move.journal_id.active:
```

Comment:
Not sure about this condition since `journal_id = None` will raise something during the creation of the move because this field is required